dkist/dataset/tiled_dataset.py

```diff
@@ -368,10 +368,12 @@ def slice_tiles(self):
             >>> ds = load_dataset(VBI_AJQWW)  # doctest: +REMOTE_DATA
             >>> ds.slice_tiles[0, 10:-10]  # doctest: +REMOTE_DATA
             <dkist.dataset.tiled_dataset.TiledDataset object at ...>
-            This VBI Dataset AJQWW is an array of (3, 3) Dataset objects and
+            This VBI Dataset is an array of (3, 3) Dataset objects and
             consists of 9 frames.
             Files are stored in ...
 
+            This calibration has Dataset ID AJQWW and the original frames have Product ID ...
```

Comment:
Is "..." a placeholder?

Reply:
Ellipsis in doc tests ignore a section of the output for matching reasons. So in this case we ignore the path so the test isn't hard coded to a specific directory